File: theme test scripts/fz.c
#include "stdio.h"
#include "string.h"

int fizzbuzz(int lim) { 
    for (int num = 1; num <= lim; num++) {
        char text[16] = "";
   	
        if (num % 3 == 0)
            strcat(text, "fizz");

        if (num % 5 == 0)
            strcat(text, "buzz");
 	
        text[0] != '\0' ? printf("%s\n", text) : printf("%d\n", num);
    }
    return 0;
}

int main(void) { 
    int lim;
    printf("Enter the limit: ");
    scanf("%d", &lim);

    fizzbuzz(100);

    return 0;
}

File: theme test scripts/fz.ml
let fizzbuzz num  =
  match num with
  | num when num mod 3 == 0 && num mod 5 == 0 -> "fizzbuzz"
  | num when num mod 3 == 0 -> "fizz"
  | num when num mod 5 == 0 -> "buzz"
  | _ -> string_of_int num;;

for n = 1 to 100 do
  print_endline (fizzbuzz n)
done